readme.md

```diff
@@ -1,151 +1,136 @@
 ![Mend Renovate CLI banner](https://docs.renovatebot.com/assets/images/mend-renovate-cli-banner.jpg)
 
-# Renovate
-
-Automated dependency updates.
-Multi-platform and multi-language.
-
 [![License: AGPL-3.0-only](https://img.shields.io/badge/license-%20%09AGPL--3.0--only-blue.svg)](https://raw.githubusercontent.com/renovatebot/renovate/main/license)
 [![codecov](https://codecov.io/gh/renovatebot/renovate/branch/main/graph/badge.svg)](https://codecov.io/gh/renovatebot/renovate)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 [![Build status](https://github.com/renovatebot/renovate/actions/workflows/build.yml/badge.svg)](https://github.com/renovatebot/renovate/actions/workflows/build.yml)
 ![Docker Pulls](https://img.shields.io/docker/pulls/renovate/renovate?color=turquoise)
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/renovatebot/renovate/badge)](https://securityscorecards.dev/viewer/?uri=github.com/renovatebot/renovate)
 
-## Why Use Renovate?
+# What is the Mend Renovate CLI?
```

Comment:
Just a quick note, we also have this issue:

- #29446

I'm assigning `@justo-mend` to that issue, and will let them decide what to do with the todos listed in that issue. :wink:

I'm mentioning this, in case you want to automatically close my issue with your revamp PR. :wink:

Reply:
OK. Thanks. I think we can work on those items in subsequent updates.
Thanks for the detailed list of ideas.